cloudformation.yaml

```diff
@@ -225,15 +259,32 @@ Resources:
                  "Next": "MigrateShards",
                  "Retry": [ {
                    "ErrorEquals": [ "States.ALL" ],
-                   "IntervalSeconds": 10,
-                   "MaxAttempts": 12
+                   "IntervalSeconds": 30,
+                   "MaxAttempts": 20,
+                   "BackoffRate": 1.0
                   } ]
                },
                "MigrateShards": {
                   "Type": "Task",
                   "Resource": "${MigrateShardsArn}",
-                  "End": true
-               }
+                  "Next": "ShardMigrationCheck"
+               },
+               "ShardMigrationCheck": {
+                  "Type": "Task",
+                  "Resource": "${ShardMigrationCheckArn}",
+                  "Next": "RemoveNode",
+                  "Retry": [ {
+                    "ErrorEquals": [ "States.ALL" ],
+                    "IntervalSeconds": 120,
```

Comment:
This interval and number of attempts was chosen fairly arbitrarily for testing purposes - any suggestions welcome!